buildenv/docker/jdk8/ppc64le/ubuntu16/Dockerfile

```diff
@@ -0,0 +1,88 @@
+# Copyright (c) 2017, 2017 IBM Corp. and others
+#
+# This program and the accompanying materials are made available under
+# the terms of the Eclipse Public License 2.0 which accompanies this
+# distribution and is available at https://www.eclipse.org/legal/epl-2.0/
+# or the Apache License, Version 2.0 which accompanies this distribution and
+# is available at https://www.apache.org/licenses/LICENSE-2.0.
+#
+# This Source Code may also be made available under the following
+# Secondary Licenses when the conditions for such availability set
+# forth in the Eclipse Public License, v. 2.0 are satisfied: GNU
+# General Public License, version 2 with the GNU Classpath
+# Exception [1] and GNU General Public License, version 2 with the
+# OpenJDK Assembly Exception [2].
+#
+# [1] https://www.gnu.org/software/classpath/license.html
+# [2] http://openjdk.java.net/legal/assembly-exception.html
+#
+# SPDX-License-Identifier: EPL-2.0 OR Apache-2.0
+
+# To use this docker file:
+#   docker build -t=openj9 .
+#   docker run -it openj9
+
+FROM ubuntu:16.04
+
+# Get prerequisites for add-apt-repository
+RUN apt-get update \
+  && apt-get install -qq -y --no-install-recommends \
+    software-properties-common \
+    python-software-properties
+
+# Add apt-get repository containing openjdk7
+RUN apt-get update \
```

Comment:
update has already been run in the previous step. Does it need to be run again?

Reply:
I know it needs to be run at least twice; once before the first apt-get (1st time), and once after we apt-add the new repository (3rd time). I forget if we need it the 2nd time. Only one way to find out.

Reply:
Ok, I just checked. Things work fine if we remove the second update. Will commit the change.